test/prompt/test_prompt_node.py

```diff
@@ -1212,3 +1212,22 @@ def test_prompt_no_truncation(mock_model, caplog):
     with caplog.at_level(logging.DEBUG):
         _ = node.prompt(PromptTemplate(prompt))
         assert prompt in caplog.text
+
+
+@pytest.mark.integration
```

Comment:
We don't really need a real model to test that the invocation_context is set as expected. Let's not use google/flan-t5-base (that's what prompt_model "hf" gets) to check this. Instead let's mock the model and make it a unit test.